HLTrigger/Configuration/python/HLT_75e33/tasks/HLTHgcalTiclPFClusteringForEgammaL1SeededTask_cfi.py

```diff
@@ -14,16 +14,19 @@
 from ..modules.ticlLayerTileProducerL1Seeded_cfi import *
 from ..modules.ticlSeedingL1_cfi import *
 from ..modules.ticlTrackstersCLUE3DHighL1Seeded_cfi import *
+from ..modules.hgcalMergeLayerClusters_cfi import hgcalMergeLayerClusters
 from ..tasks.HLTBeamSpotTask_cfi import *
 
+hgcalMergeLayerClustersL1Seeded = hgcalMergeLayerClusters.clone()
```

Comment:
Is `hgcalMergeLayerClustersL1Seeded` really supposed to be a copy of `hgcalMergeLayerClusters` ? Shouldn't they differ because the first one is restricted to L1-seeded clusters ?

Reply:
This wasn't addressed. Shouldn't `hgcalMergeLayerClustersL1Seeded` have its own cfi and call `hgcalLayerClusters*L1Seeded` in its InputTags ?

Reply:
yes it should, thank you. It is fixed now